tools/permuter_settings.toml

```diff
@@ -1,3 +1,5 @@
+compiler_type = "ido"
```

Comment:
What does this do/change?

Reply:
It's a new permuter option that adjusts weightings and which passes to run. It complains if it's not present.